cmd/state-transition/expected_state.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+   Copyright 2022 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "expected_state.hpp"
+
+#include <iostream>
+
+#include <nlohmann/json.hpp>
+
+#include "silkworm/core/chain/config.hpp"
+#include "silkworm/core/common/test_util.hpp"
+
+namespace silkworm::cmd::state_transition {
+
+silkworm::ChainConfig ExpectedState::get_config() const {
```

Comment:
Fully-qualified type name `silkworm::ChainConfig` can be simplified to just `ChainConfig` when we're already within `silkworm` namespace (this applies also to other types and namespaces in some other places)